_sass/reuse.scss

```diff
@@ -0,0 +1,18 @@
+#services {
+  h2 {
+      max-width: 60rem;
```

Comment:
The [tutorial](http://webdesign.tutsplus.com/tutorials/comprehensive-guide-when-to-use-em-vs-rem--cms-23984) I sent you last time didn’t make it clear enough to you?